### ISSUE TITLE
Add official promo cards

### ISSUE THREAD

**Issue by alrusdi** [bafolts/terraforming-mars#539]
We are missing these:

- [x] Penguins
- [x] Small Asteroid
- [ ] Self-Replicating robots
- [x] Snow Algae
- [x] Dusk Laser Mining
- [x] Project Inspection
- [x] Energy Market
- [x] Hi-Tech Lab
- [x] Interplanetary Trade
- [x] Law Suit
- [x] Mercurian Alloys
- [x] Orbital Cleanup
- [x] Rego Plastics
- [x] Saturn Surfing
- [x] Stanford Torus

Political Alliance (Turmoil)

**Comment by vincentneko** [comment]
Penguins, Small Asteroid, Self-Replicating robots and Snow Algae missing as well.

https://ssimeonoff.github.io/cards-list 
with promo filter will display all these cards

**Comment by alrusdi** [comment]
> Penguins, Small Asteroid, Self-Replicating robots and Snow Algae missing as well.

Indeed. I found their name in CardName enum, but there no real cards for them :-)

**Comment by gomfunkel** [comment]
@alrusdi @vincentneko If no one started this yet feel free to assign it to me. Should find some time to add the cards soonish.

**Comment by alrusdi** [comment]
@gomfunkel  It would be great. Are you ready to write unit tests for it? :-)

**Comment by gomfunkel** [comment]
@alrusdi Sure thing. No cards without tests. I might even write the tests before implementing the cards... 😉

**Comment by alrusdi** [comment]
You are welcome! Do it one by one to make reviews easy

**Comment by gomfunkel** [comment]
@alrusdi Do you prefer one pull request per card or one commit per card?

**Comment by alrusdi** [comment]
One pull request per card will be awesome!

**Comment by alrusdi** [comment]
@gomfunkel great work! But can you see now why i requested to PR cards one by one? Unfortunately, the card with a bug is on the top of the stack :-(

**Comment by alrusdi** [comment]
@gomfunkel thank you very much for this great work!
I think "Self replicating robots" is too complicated for our version of the game. We can live without it.

**Comment by gomfunkel** [comment]
@alrusdi You're welcome! 

We should create a new issue for _Political Alliance_ which can be implemented once Turmoil is finished or along with the Turmoil cards. 

Regarding _Self-Replicating Robots_: Yes, that's a complicated one. I do have an idea on how to implement the card and will have a try at it. Can't promise anything, though.
